app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(string: str) -> None:
+    names = string.split(" ")
+    if len(names) >= 3 and names[0] == "cp":
+        source_file, dest_file = names[1], names[2]
```

Comment:
Unpack the whole variable `names` without using indexes, e.g.:
```suggestion
       command, source_filename, destination_filename = names
```